venter/tests.py

```diff
@@ -273,13 +273,25 @@ def test_send_mass_mail(self):
         queryset = Complaints.objects.filter(status=STATUS_REPORTED)
         complaint_admin.send_emails(complaint_admin, request, queryset)
         self.assertEqual(len(mail.outbox), 1)
-        self.assertEqual(len(queryset.filter(email_status=True)), 1)
 
         queryset = Complaints.objects.filter(status=STATUS_IN_PROGRESS)
         complaint_admin.send_emails(complaint_admin, request, queryset)
         self.assertEqual(len(mail.outbox), 2)
-        self.assertEqual(len(queryset.filter(email_status=True)), 1)
+
+        # Test for email_sent_to list
+        complaint_multival = Complaints.objects.create(created_by=self.user.profile, description='multiple')
+        new_auth3 = Authorities.objects.create(email='auth3@example.com', name='auth3')
+        complaint_multival.authorities.add(new_auth3)
+        queryset = Complaints.objects.filter(description='multiple')
+        complaint_admin.send_emails(complaint_admin, request, queryset)
+        self.assertEqual(len(queryset.filter(email_sent_to='auth3')), 1)
```

Comment:
This seems somewhat contrived. Won't just asserting the `email_sent_to` of `complaint_multival` to `auth3` work? Similar below.

Reply:
This wouldn't work since we appending the names of the authorities to the email_sent_to field

Reply:
What? IIUC, here's what this code is doing:
* L285 - `queryset` has 1 item, that's `complaint_multival`
* L286 - send the emails. `queryset` remains the same, `email_sent_to` for `complaint_multival` is updated to "auth3".
* L287 - filter `queryset` by `email_sent_to`, and it still has one item, which is _still_ `complaint_multival`
So, everything else remaining same, how would checking `email_sent_to` explicitly on L287 change anything?

Reply:
Ok, I think I understand what you mean. I'll make the necessary changes